interfaces/Portalicious/src/app/pages/project-registration-personal-information/project-registration-personal-information.page.ts

```diff
@@ -62,20 +69,52 @@ export class ProjectRegistrationPersonalInformationPageComponent {
     ),
   );
 
+  project = injectQuery(this.projectApiService.getProject(this.projectId));
+
   attributeList = computed<DataListItem[]>(() => {
     const list: DataListItem[] = [];
     if (!this.projectAttributes.isSuccess() || !this.registration.isSuccess()) {
       return list;
     }
 
+    const genericAttributeNames =
```

Comment:
A side effect of getting the genericAttributeNames seperate is that they always appears at the start of the data list now. Which I think is nice.